prisma/schema.prisma

```diff
@@ -221,7 +223,6 @@ model match {
   id                       Int               @id(map: "PK_92b6c3a6631dd5b24a67c69f69d") @default(autoincrement())
   uuid                     String            @unique(map: "IDX_65a3ec8c0aa6c3c9c04f5b53e3") @db.VarChar
   dissolved                Boolean           @default(false)
-  dissolvedAt              DateTime?         @db.Timestamp(6)
```

Comment:
@Jonasdoubleyou why has this been removed? Seems like a mistake honestly.

Reply:
jup, wrong merge from master